src/libxtp/qmpackages/orca.h

```diff
@@ -57,7 +57,29 @@ class Orca : public QMPackage {
 
   Eigen::Matrix3d GetPolarizability() const override;
 
+ protected:
+  const std::array<Index, 25>& ShellMulitplier() const final {
+    return _multipliers;
+  }
+  const std::array<Index, 25>& ShellReorder() const final { return _reorder; }
+
  private:
+  // clang-format off
+  std::array<Index,25> _multipliers={
```

Comment:
Could you please explain to me what are those multipliers?

Reply:
to be honest I do not really get it myself. In my mind they should not be there but @baumeier found that without them it does not work. @baumeier want to explain?

Reply:
Not much to explain. At some point, when we compared the coefficients coming out of orca to other codes (and our own), some entries associated to these functions showed a minus sign difference. BUT only on some elements, not the whole vector. 
Orca is using libint, right? Maybe one can look into that to see if there is a difference in definition of the affected AOs? @felipeZ don‘t you have some experience with libint?

Reply:
it could actually be a different normalisation condition.

Reply:
Also possible, but orca is a bit too black boxy overall. I don‘t think they have these details documented.

Reply:
Nor do we...

Reply:
it is in the code :)

Reply:
...somewhere

Reply:
@baumeier  I have experience with libint2, I use it to compute multipole integrals using CP2K optimized basissets